// ngx-videogular@13.2.4/esm2020/lib/ima-ads/ima-ads.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { VgImaAds } from './vg-ima-ads';
import * as i0 from "@angular/core";
export class VgImaAdsModule {
}
VgImaAdsModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgImaAdsModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
VgImaAdsModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgImaAdsModule, declarations: [VgImaAds], imports: [CommonModule], exports: [VgImaAds] });
VgImaAdsModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgImaAdsModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgImaAdsModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [
                        VgImaAds
                    ],
                    exports: [
                        VgImaAds
                    ]
                }]
        }] });
