// ngx-videogular@13.2.4/esm2020/lib/core/vg-core.module.mjs
import { NgModule } from '@angular/core';
import { VgControlsHidden } from './services/vg-controls-hidden';
import { VgFullscreenAPI } from './services/vg-fullscreen-api';
import { VgUtils } from './services/vg-utils';
import { VgAPI } from './services/vg-api';
import { VgMedia } from './vg-media/vg-media';
import { VgCuePoints } from './vg-cue-points/vg-cue-points';
import { VgPlayer } from './vg-player/vg-player';
import { VgStates } from './states/vg-states';
import { VgEvents } from './events/vg-events';
import * as i0 from "@angular/core";
export * from './vg-media/i-media-element';
export * from './vg-media/i-playable';
export * from './vg-media/vg-media-element';
export * from './services/vg-api';
export * from './services/vg-fullscreen-api';
export * from './services/vg-utils';
export * from './services/vg-controls-hidden';
export * from './events/vg-events';
export * from './states/vg-states';
export * from './vg-media/vg-media';
export * from '../buffering/vg-buffering';
export * from '../controls/vg-controls';
export * from '../controls/vg-fullscreen/vg-fullscreen';
export * from '../controls/vg-mute/vg-mute';
export * from '../controls/vg-volume/vg-volume';
export * from '../controls/vg-play-pause/vg-play-pause';
export * from '../controls/vg-playback-button/vg-playback-button';
export * from '../controls/vg-scrub-bar/vg-scrub-bar';
export * from '../controls/vg-scrub-bar/vg-scrub-bar-buffering-time/vg-scrub-bar-buffering-time';
export * from '../controls/vg-scrub-bar/vg-scrub-bar-cue-points/vg-scrub-bar-cue-points';
export * from '../controls/vg-scrub-bar/vg-scrub-bar-current-time/vg-scrub-bar-current-time';
export * from '../controls/vg-time-display/vg-time-display';
export * from '../controls/vg-track-selector/vg-track-selector';
export * from '../controls/vg-quality-selector/vg-quality-selector';
export * from '../ima-ads/vg-ima-ads';
export * from '../overlay-play/vg-overlay-play';
export * from './vg-cue-points/vg-cue-points';
export * from './vg-player/vg-player';
export class VgCoreModule {
}
VgCoreModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgCoreModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
VgCoreModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgCoreModule, declarations: [VgMedia, VgCuePoints, VgPlayer], exports: [VgMedia, VgCuePoints, VgPlayer] });
VgCoreModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgCoreModule, providers: [VgAPI, VgFullscreenAPI, VgUtils, VgControlsHidden, VgStates, VgEvents] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgCoreModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [VgMedia, VgCuePoints, VgPlayer],
                    exports: [VgMedia, VgCuePoints, VgPlayer],
                    providers: [VgAPI, VgFullscreenAPI, VgUtils, VgControlsHidden, VgStates, VgEvents]
                }]
        }] });
