// ngx-videogular@13.2.4/esm2020/lib/overlay-play/vg-overlay-play.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { VgOverlayPlay } from './vg-overlay-play';
import * as i0 from "@angular/core";
export class VgOverlayPlayModule {
}
VgOverlayPlayModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgOverlayPlayModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
VgOverlayPlayModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgOverlayPlayModule, declarations: [VgOverlayPlay], imports: [CommonModule], exports: [VgOverlayPlay] });
VgOverlayPlayModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgOverlayPlayModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgOverlayPlayModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [
                        VgOverlayPlay
                    ],
                    exports: [
                        VgOverlayPlay
                    ]
                }]
        }] });
