// ngx-videogular@13.2.4/esm2020/lib/streaming/vg-streaming.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { VgDASH } from './vg-dash/vg-dash';
import { VgHLS } from './vg-hls/vg-hls';
import * as i0 from "@angular/core";
export * from './vg-dash/vg-dash';
export * from './vg-hls/vg-hls';
export class VgStreamingModule {
}
VgStreamingModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgStreamingModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
VgStreamingModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgStreamingModule, declarations: [VgDASH, VgHLS], imports: [CommonModule], exports: [VgDASH, VgHLS] });
VgStreamingModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgStreamingModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgStreamingModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [
                        VgDASH, VgHLS
                    ],
                    exports: [
                        VgDASH, VgHLS
                    ]
                }]
        }] });
