// ngx-videogular@13.2.4/esm2020/lib/buffering/vg-buffering.module.mjs
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { VgBuffering } from './vg-buffering';
import * as i0 from "@angular/core";
export class VgBufferingModule {
}
VgBufferingModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgBufferingModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
VgBufferingModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgBufferingModule, declarations: [VgBuffering], imports: [CommonModule], exports: [VgBuffering] });
VgBufferingModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgBufferingModule, imports: [[CommonModule]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.2", ngImport: i0, type: VgBufferingModule, decorators: [{
            type: NgModule,
            args: [{
                    imports: [CommonModule],
                    declarations: [
                        VgBuffering
                    ],
                    exports: [
                        VgBuffering
                    ]
                }]
        }] });
